### ISSUE TITLE
dino fails to build: qrencode not found (missing cmake check)

### ISSUE THREAD

**Issue by RalfJung** [dino/dino#454]
Dino fails to build on my system, saying
```
plugins/omemo/src/contact_details_dialog.c:16:10: fatal error: qrencode.h: No such file or directory
 #include "qrencode.h"
          ^~~~~~~~~~~~
```
This is despite the fact that cmake completed successfully.

Seems like the cmake file was not updated to reflect the dependency on qrencode?

**Comment by RalfJung** [comment]
> libqrencode-dev is needed.

Yes, and the cmake files fail to check for it.

A quick grep through the source shows no sign of cmake even trying to check for this header.

> Is this a clean build or git pull?

`git pull`. I am not going to re-configure all cmake flags after every pull.

**Comment by RalfJung** [comment]
But FWIW, I did a clean build (`mv build build.old && ./configure && make`), same result.

**Comment by fiaxh** [comment]
Fixed in cdd4c0b .
